<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mdtSqlCopierCsvFileImportDialog</class>
 <widget class="QDialog" name="mdtSqlCopierCsvFileImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>992</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSV file import</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="gbDestimationDatabase">
     <property name="title">
      <string>Destination database</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QToolButton" name="tbSelectDestinationDatabase">
        <property name="statusTip">
         <string>Select database</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset theme="document-open">
          <normaloff>../../../../</normaloff>../../../../</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="mdtSqlDatabaseBasicInfoWidget" name="wDestinationDatabaseInfo" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbMapping">
     <property name="title">
      <string>Mapping</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="tbRemoveCopyItem">
          <property name="text">
           <string>Remove</string>
          </property>
          <property name="icon">
           <iconset theme="list-remove">
            <normaloff/>
           </iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbEditCopyItem">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbAddCopyItem">
          <property name="text">
           <string>Add</string>
          </property>
          <property name="icon">
           <iconset theme="list-add">
            <normaloff/>
           </iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tvMapping"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pbImport">
       <property name="text">
        <string>Import</string>
       </property>
       <property name="icon">
        <iconset theme="system-run">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbAbort">
       <property name="text">
        <string>Abort</string>
       </property>
       <property name="icon">
        <iconset theme="process-stop">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="pbGlobalProgress">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mdtSqlDatabaseBasicInfoWidget</class>
   <extends>QWidget</extends>
   <header>mdtSqlDatabaseBasicInfoWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>mdtSqlCopierCsvFileImportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>mdtSqlCopierCsvFileImportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
